specification/schemas/BaseCustoms.json

```diff
@@ -38,19 +38,14 @@
     "shipping_value": {
       "oneOf": [
         {
-          "allOf": [
-            {
-              "$ref": "#/components/schemas/Price"
-            },
-            {
-              "description": "The shipping cost paid by the customer, for tax and duty purposes."
-            }
-          ]
+          "$ref": "#/components/schemas/Price"
         },
         {
-          "type": "null"
+          "type": "null",
+          "description": ""
         }
-      ]
+      ],
+      "description": "The shipping cost paid by the customer, for tax and duty purposes."
```

Comment:
Why not the description up there in the previous field that gets merged?

Reply:
Because then Redoc does not show it when the item is collapsed. This is the only way I got it to show nicely, but it involved having an empty description for the null, other wise it would show twice 💩